.zenodo.json

```diff
@@ -0,0 +1,59 @@
+{
+    "creators":[
+        {
+            "name": "SuperDARN Data Analysis Working Group"
+        },
+        {
```

Comment:
Do we always do alphabetically? Sorry in pyDARN I asked once and someone sad put 2 more prominent developers first then the rest should be alphabetic.

Reply:
I think this is how it was done for v4.3 so I figured I'd follow suit.  This is something we wanted to bring up during the last meeting about authorship, but didn't get much time for it.  So, I'll defer until we can get something figured out then as a group.

Reply:
The author list for v4.3 was not done alphabetically but in some vague order of Github contributions.  Earlier releases started with myself and Pasha and then continued alphabetically.  (not advocating for any particular convention, just providing context)

Reply:
Thanks for providing the context here @egthomas and sorry I had it wrong.  I'm guessing the Github contributions is how Zenodo tried to figure out the order.